tools/launchdev.sh

```diff
@@ -65,6 +68,76 @@ function init(){
         echo "Config file $ST2_CONF does not exist."
         exit 1
     fi
+
+    # Optionally, initialize mistral
+    if [ "${include_mistral}" = true ]; then
+        init_mistral
+    fi
+}
+
+function init_mistral(){
+    # Both the mistral and st2mistral repos must be present alongside the st2 repo
+    MISTRAL_REPO="${ST2_REPO}/../mistral"
+    ST2MISTRAL_REPO="${ST2_REPO}/../st2mistral"
+
+    if [ ! -d "$MISTRAL_REPO" ] || [ ! -d "$ST2MISTRAL_REPO" ] ; then
+        echo "You specified the Mistral option, but either the mistral or st2mistral directories were not found."
+        echo "Please place a clone of both mistral and st2mistral repositories alongside the st2 repository."
+        exit 1
+    fi
+
+    if [ -z "$MISTRAL_CONF" ]; then
+        MISTRAL_CONF=${ST2_REPO}/conf/mistral.dev/mistral.dev.conf
+    fi
+    echo "Using mistral config file: $MISTRAL_CONF"
+
+    if [ ! -f "$MISTRAL_CONF" ]; then
+        echo "Config file $MISTRAL_CONF does not exist."
+        exit 1
+    fi
+
+    # Delete mistral database
+    # (May look into doing something a bit more fancy in the future so we don't have to recreate mistral database each time,
+    # but this works for now)
+    st2stop
+    rm "${ST2_REPO}/mistral.db"
+
+    # Initialize mistral database if it doesn't already exist
+    if [ ! -f "${ST2_REPO}/mistral.db" ]; then
+        echo "Initializing Mistral database. This will take some time because tox needs to download deps. Please be patient..."
+
+        # We're using the StackStorm venv for everything
+        # TODO (mierdin): Evaluate if it's worth using a separate venv for installing and running mistral
+        source "${ST2_REPO}/virtualenv/bin/activate"
```

Comment:
Can you put mistral requirements in its own venv? Please follow https://github.com/StackStorm/mistral_dev/blob/master/actions/install.sh.